platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/files/hashmap/HalfDiskHashMap.java

```diff
@@ -481,7 +513,9 @@ public long get(final K key, final long notFoundValue) throws IOException {
         final int bucketIndex = computeBucketIndex(keyHash);
         final Bucket<K> bucket = fileCollection.readDataItemUsingIndex(bucketIndexToBucketLocation, bucketIndex);
         if (bucket != null) {
-            return bucket.findValue(keyHash, key, notFoundValue);
+            try (bucket) {
```

Comment:
A little unusual usage, might be clearer written as follows:
```java
        try (final Bucket<K> bucket = fileCollection.readDataItemUsingIndex(bucketIndexToBucketLocation, bucketIndex)) {
            if(bucket != null)
                return bucket.findValue(keyHash, key, notFoundValue);
            else
                return notFoundValue;
        }
```

Reply:
Agree, it looks clearer. Let me change it this way.